### PR TITLE
Featured deck moderator panel

### PR DESCRIPTION
**Pull request by alexmiddeleer**

### CODE REVIEW

**Comment by alexmiddeleer** [comment]
If possible, testing this real quick would be great - but not critical if you don't have time.

**Comment by lily-pytel** [comment]
Crazy crunch time at work so I won't be able to at least until the weekend :-(

**Comment by alexmiddeleer** [comment]
@lily-pytel No problem, sorry to hear that. Hope things get better :(

File: next/components/moderator-featured-decks-editor.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+import { useState } from 'react';
```
Comment (lily-pytel) [inline next/components/moderator-featured-decks-editor.js]: I'm surprised the linter didn't yell about multiple react declarations 😆

File: next/components/moderator-featured-decks-editor.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+import {
+  deckFeaturedQuery,
+  deleteFeaturedDeckMutation,
+  addFeaturedDeckMutation
+} from '../lib/deck-queries.js';
+import { handleInputChangeHooks } from '../lib/form-utils.js';
+
+function moderatorFeaturedDecksEditor() {
```
Comment (lily-pytel) [inline next/components/moderator-featured-decks-editor.js]: Nbd obviously but all our other components have been Pascal case, and I think that's the standard for react components?

**Pull request merged**